Packs/GCP-IAM/Integrations/GCPIAM/GCPIAM.py

```diff
@@ -2919,6 +2948,40 @@ def gcp_iam_service_account_key_disable_command(client: Client, args: Dict[str,
     return command_results_list
 
 
+def gcp_iam_service_account_generate_access_token_command(client: Client, args: Dict[str, Any]) -> list:
+    """
+    Create a serivce account short-lived access token
+
+    Args:
+        client (Client): GCP API client.
+        args (dict): Command arguments from XSOAR.
+
+    Returns:
+        list[CommandResults]: outputs, readable outputs and raw response for XSOAR.
```

Comment:
Seems to return just a single `CommandResults` and not a list?
If so -
```suggestion
        CommandResults: outputs, readable outputs and raw response for XSOAR.
```